css/menu.css

```diff
@@ -415,3 +415,7 @@ html.no-history .fl-viewport-header .nav-left {
 .has-overlay-menu {
   overflow-y: hidden;
 }
+
+.hidden {
```

Comment:
Why is this needed?

![image](https://user-images.githubusercontent.com/290733/109194115-02733500-7791-11eb-84dd-5b010a5afb7e.png)

Bootstrap already has this exact same style.

Reply:
@tonytlwu When I tried simply add a hidden class it has no effect. Should Bootrap do be added as a dependency?

Reply:
Bootstrap should be available to all pages already. Please check and confirm why this is needed.